public/javascripts/SVValidate/src/status/StatusExample.js

```diff
@@ -57,12 +57,15 @@ function StatusExample (statusUI) {
     }
 
     /**
-     * Sets the horizontal position and height of the popup based on which picture was hovered over.
+     * Sets the horizontal position, vertical position, and height of the popup
+     * as well as the popup pointer basedon which picture was hovered over.
```

Comment:
Can this comment be made more concise so that it fits on one line (max 120 characters)?